platform/marvell-armhf/sonic-platform-nokia/7215/sonic_platform/watchdog.py

```diff
@@ -127,15 +142,20 @@ def arm(self, seconds):
         ret = WD_COMMON_ERROR
         if seconds < 0:
             return ret
-
+        
+        # Stop the watchdog service to gain access of watchdog file pointer
+        if self.is_armed():
+            os.popen("systemctl stop cpu_wdt.service")
+            time.sleep(2)
+        if not self.watchdog:
+            self.watchdog = os.open(self.watchdog_path, os.O_WRONLY)
         try:
             if self.timeout != seconds:
                 self.timeout = self._settimeout(seconds)
-            if self.armed:
```

Comment:
@Pavan-Nokia please update the test result for disarm as well. i see only arming result in PR description.

Reply:
added additional test results and logs in description.